modules/mobile/spec/requests/mobile/v0/health/observations_spec.rb

```diff
@@ -61,7 +61,7 @@
         get '/mobile/v0/health/observations/I2-ILWORI4YUOUAR5H2GCH6ATEFRM000000', headers: sis_headers
       end
     end
-
+    expect(PersonalInformationLog.count).to eq(0)
```

Comment:
i did test locally that altering the schema of the upstream response does increment this by 1.